cardano-cli/src/Cardano/CLI/Legacy/Commands/Pool.hs

```diff
@@ -47,7 +47,10 @@ data LegacyPoolCmds
       EpochNo
       -- ^ Epoch in which to retire the stake pool.
       (File () Out)
-  | PoolGetId (VerificationKeyOrFile StakePoolKey) PoolIdOutputFormat (Maybe (File () Out))
+  | PoolGetId
```

Comment:
Don't add anything else to the legacy commands. We only want to maintain them. Anything new should only go under the new era based commands.